listenbrainz/webserver/views/stats_api.py

```diff
@@ -345,6 +346,86 @@ def get_recording(user_name):
     }})
 
 
+@stats_api_bp.route("/user/<user_name>/listening_activity")
```

Comment:
```suggestion
@stats_api_bp.route("/user/<user_name>/listening-activity")
```

We generally use dashes in API endpoints